### PR TITLE
Extension sample for Debugging Extensions tutorial

### PR DESCRIPTION
**Pull request by AmySteam**
This PR updates the MV2 extension "Change background color" sample to MV3. 
This extension will be used to teach developers about [Debugging extensions](https://developer.chrome.com/docs/extensions/mv3/tut_debugging/).

### CODE REVIEW

File: functional-samples/tutorial.broken-color/popup.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Google LLC
```
Comment (sebastianbenz) [inline functional-samples/tutorial.broken-color/popup.js:1]: 2023?

File: functional-samples/tutorial.broken-color/popup.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const changeColorButton = document.getElementById('changeColor');
+
+// Retrieve the color from storage and update the button's style and value
+chrome.storage.sync.get('color', ({ color }) => {
+  changeColorButton.style.backgroundColor = color;
+  changeColorButton.setAttribute('value', color);
+});
+
+// Update the click event handler with an arrow function
```
Comment (sebastianbenz) [inline functional-samples/tutorial.broken-color/popup.js:23]: What does "arrow function" mean here?

File: functional-samples/tutorial.broken-color/service-worker.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// There's a typo in the line below;
+// ❌ oninstalled should be ✅ onInstalled.
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.storage.sync.set({ color: '#3aa757' }, () => {
+    console.log('The color is green.');
```
Comment (sebastianbenz) [inline functional-samples/tutorial.broken-color/service-worker.js:19]:
```suggestion
    console.log('The background color is green.');
```

**Review by sebastianbenz — APPROVED** [review]
Looks good! Two small nits.

File: functional-samples/tutorial.broken-color/popup.js (hunk 1 above)
Comment (AmySteam) [reply functional-samples/tutorial.broken-color/popup.js:1]: 😅 Whoops missed that

File: functional-samples/tutorial.broken-color/popup.js (hunk 2 above)
Comment (AmySteam) [reply functional-samples/tutorial.broken-color/popup.js:23]: Removing

**Pull request merged**